cmd/influx/authorization.go

```diff
@@ -299,23 +287,13 @@ func authFindCmd() *cobra.Command {
 
 	cmd.Flags().StringVarP(&authorizationFindFlags.user, "user", "u", "", "The user")
 	cmd.Flags().StringVarP(&authorizationFindFlags.userID, "user-id", "", "", "The user ID")
-	cmd.Flags().StringVarP(&authorizationFindFlags.org, "org", "o", "", "The org")
-	viper.BindEnv("ORG")
-	if h := viper.GetString("ORG"); h != "" {
-		authorizationFindFlags.org = h
-	}
 
-	cmd.Flags().StringVarP(&authorizationFindFlags.orgID, "org-id", "", "", "The org ID")
-	viper.BindEnv("ORG_ID")
-	if h := viper.GetString("ORG_ID"); h != "" {
-		authorizationFindFlags.orgID = h
-	}
 	cmd.Flags().StringVarP(&authorizationFindFlags.id, "id", "i", "", "The authorization ID")
 
 	return cmd
 }
 
-func newAuthorizationService(f Flags) (platform.AuthorizationService, error) {
+func newAuthorizationService() (platform.AuthorizationService, error) {
```

Comment:
I am never a big fan of anything global. If passing the flags bothers you, you could compose a struct and make this function a method.

Reply:
totally agree. I'd like to see all the commands use the builder pattern of sorts so we can dep inject. We'r enot quite there yet. That'll take some time.